src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -148,50 +151,13 @@ newrelic-infrastructure:
 ```
 
 <Callout variant="tip">
-  Notice we are specifying `--tls --port 10250`. Earlier versions of the integration might have worked without this, as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3) of the Kubernetes integration it is mandatory to specify them. The reason for this change is that the integration now connects to the Kubelet using the nodeIP rather than `localhost`, with the former requiring TLS while the latter did not.
+  Notice we are specifying `--tls --port 10250`. Earlier versions of the integration might have worked without this, as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3) of the Kubernetes integration it is mandatory to specify them. The reason for this change is that the integration now connects to the Kubelet using the nodeIP rather than `localhost`, with the former requiring TLS while the latter did not.
```

Comment:
```suggestion
 Note that we're specifying `--tls --port 10250` in the command. While previous versions of the integration might have functioned without these flags, they are now mandatory as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3). This change is due to the integration's connection method shifting from `localhost` to the nodeIP, which requires TLS encryption.
```